http/middleware_test.go

```diff
@@ -0,0 +1,67 @@
+package http
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWithHeaderAuthentication_InvalidAuthorization(t *testing.T) {
```

Comment:
❓ Don't you need a successful test case as well?

Reply:
Yes, setting up tests for token verification is cumbersome, but I'll get back to this. I'll open a ticket to track this.